edge/pkg/metamanager/metaserver/server.go

```diff
@@ -168,21 +121,18 @@ func (ls *MetaServer) startHTTPServer(stopChan <-chan struct{}) {
 	beehiveContext.Cancel()
 }
 
-func (ls *MetaServer) startHTTPSServer(stopChan <-chan struct{}) {
-	_, err := ls.getCurrent()
+func (ls *MetaServer) startHTTPSServer(addr string, stopChan <-chan struct{}) {
```

Comment:
The private function can returns an error to the caller for processing.

Reply:
> The private function can returns an error to the caller for processing.

`startHTTPSServer` will be called in the form of a goroutine, and it belongs the start process of metaserver, so I add a panic in this func.